### PR TITLE
[bitnami/elasticsearch] Fix get_machine_ip to handle IPv6 addresses

### LINKED ISSUE #67081: [bitnami/elasticsearch] SingleNode healthcheck failing on IPv6 environments
### Name and Version

bitnami/elasticsearch:8.8.1-debian-11-r0

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Configure deployment with masterOnly set to true
2. Disable data, ingest and coordinating roles deployment
3. Deploy bitnami elastic chart on pure IPv6 cluster
4. Healthceck of readiness probe will fail

### What is the expected behavior?

Healthcheck passes and container comes up.

### What do you see instead?

`  Warning  Unhealthy  26m                    kubelet            Readiness probe failed: ^[[38;5;6melasticsearch ^[[38;5;5m07:30:53.29 ^[`

### Additional information

Originally raised at https://github.com/bitnami/charts/issues/22880

### PR DESCRIPTION
**Pull request by rgaduput**
### Description of the change

Update get_machine_ip to add brackets incase of IPv6 addresses.

### Benefits

It will fix the Elasticsearch health/readiness probe failures in IPv6 only cluster deployment with single node.

### Applicable issues

- fixes #67081

### CODE REVIEW

File: bitnami/elasticsearch/8/debian-12/prebuildfs/opt/bitnami/scripts/libnet.sh [hunk 1]
```diff
@@ -68,7 +68,13 @@ get_machine_ip() {
         error "Could not find any IP address associated to hostname ${hostname}"
         exit 1
     fi
-    echo "${ip_addresses[0]}"
+
+    # Check if the first IP address is IPv6 and add brackets
+    if [[ "${ip_addresses[0]}" =~ : ]]; then
```
Comment (dgomezleon) [inline bitnami/elasticsearch/8/debian-12/prebuildfs/opt/bitnami/scripts/libnet.sh:73]: You could try `validate_ipv6` function by loading `. /opt/bitnami/scripts/libvalidations.sh` at the beginning of the file:

```suggestion
    if validate_ipv6 "${ip_addresses[0]}"; then
```

**Review by dgomezleon — COMMENTED** [review]
Hi @rgaduput , thanks for contributing

Can you try my suggestion?

**Comment by rgaduput** [comment]
> Hi @rgaduput , thanks for contributing
> 
> Can you try my suggestion?

Hi @dgomezleon thanks, did not knew that such function already exists. Updated the code as suggested, please check now.

**Review by dgomezleon — APPROVED** [review]
LGTM

**Pull request merged**